R/dominodata.R

```diff
@@ -0,0 +1,3 @@
+#' @import arrow
+#' @importFrom methods is
+NULL
```

Comment:
It's good practice to always have a newline at the end of files. I'd suggest configuring your IDE to do that automatically when saving a file